packages/cli/src/extract_subcommands.ts

```diff
@@ -0,0 +1,12 @@
+import { Argv } from 'yargs';
+
+/**
+ * If the parser finished processing arguments, attempt extracting subcommand information.
+ */
+export const extractSubCommands = (yargs: Argv): string => {
+  try {
+    return yargs.parsed ? yargs.parsed.argv._.join(' ') : '';
+  } catch {
+    return '';
+  }
```

Comment:
Under what cirucumstances this throws? Can we write checks instead? If not then we should at least assert that exception we get is expected.

Reply:
From what I saw, this is should never throw. Yargs could release a breaking change to information returned in `parsed`.